lib/Log/Any/Adapter/DERIV.pm

```diff
@@ -354,87 +386,117 @@ The log data.
 
 =back
 
-Return: processed data
-
+Returns processed data
 
 =cut
 
 sub _filter_stack {
     my ($self, $data) = @_;
-    return $data if(numeric_level($data->{severity}) <= numeric_level('warn'));
+
+    return $data if (numeric_level($data->{severity}) <= numeric_level('warn'));
+
     # now severity > warn
     return $data if $self->{log_level} >= numeric_level('debug');
+
     delete $data->{stack};
     return $data;
 }
 
 =head2 _collapse_future_stack
 
-The future frames are too much and too tedious. This method will keep only one frame if there are many continuously future frames.
-Parameter: log data
-Return: log data
+The future L<FUTURE> frames are too much and too tedious. This method will keep only one
```

Comment:
There is no `FUTURE` module... https://metacpan.org/pod/FUTURE

I'd suggest rewriting this, best discuss with the team to apply wording that makes a bit more sense!

Reply:
I think the original pod for this method was talking about https://metacpan.org/pod/Future